openjdk.test.modularity/build.xml

```diff
@@ -0,0 +1,363 @@
+							<sequential>
+								<if>
+									<contains string="@{platform}" substring="${this_platform}" />
+									<then>
+										<echo message="Creating library directory: ${jlink-nativedir}/bin/@{release}/@{platform}"/>
+										<mkdir dir="${jlink-nativedir}/bin/@{release}/@{platform}"/>
+										<!-- Make sure other users will be able to write into the directory -->
+										<!--
+										<exec executable="chmod">
+												<arg value="-fR"/>
+												<arg value="755"/>
+												<arg value="${jlink-nativedir}"/>
+										</exec>
+										-->
+										<for list="${native_src}" param="testname">
+											<sequential>
+												<echo message="${module} build native: Building test @{testname}" />
+												<var name="remote_jlink-nativedir" value="${target_root}/javasvt/test.modularity/src/tests/com.test.jlink/native"/>
+												<var name="remote_srcdir" value="${remote_jlink-nativedir}"/>
+												<var name="remote_libdir" value="../../../../${jlink-nativedir}/lib/@{release}"/>
+												<var name="remote_bindir" value="../../../../${jlink-nativedir}/bin/@{release}"/>
+												<var name="remote_headerdir" value="${remote_libdir}"/>
+												<var name="command" value='${setup_local_build_env_@{platform}_@{release}} ${gmake_@{platform}} -C ${remote_srcdir} SRC=@{testname} PLATFORM=@{platform} JAVA_HOME=${java_home_@{platform}_@{release}} OUTDIR=${remote_bindir} HEADERDIR=${remote_headerdir}'/>
+												<tempfile property="command_file_@{platform}_@{release}" destDir="${java.io.tmpdir}" prefix="jtctest.build.command."/>
+												<if>
+													<isset property="rtb.is_windows"/>
+													<then>
+														<echo message="Writing ${command} to ${command_file_@{platform}_@{release}}.bat"/>
+														<echo message="${command}" file="${command_file_@{platform}_@{release}}.bat"/>
+														<echo message="Running ${command_file_@{platform}_@{release}}.bat"/>
+														<exec executable="${command_file_@{platform}_@{release}}.bat" failonerror="true"/>
+														<delete file="${command_file_@{platform}_@{release}}.bat" verbose="true"/>
+													</then>
+													<else>
+														<echo message="Writing ${command} to ${command_file_@{platform}_@{release}}.sh"/>
+														<echo message="${command}" file="${command_file_@{platform}_@{release}}.sh"/>
+														<echo message="Running /bin/sh ${command_file_@{platform}_@{release}}.sh"/>
+														<exec executable="/bin/sh" failonerror="true">
+															<arg value="${command_file_@{platform}_@{release}}.sh"/>
+														</exec>
+														<delete file="${command_file_@{platform}_@{release}}.sh" verbose="true"/>
+													</else>
+												</if>
+											</sequential>
+										</for>
+									</then>
+									<else>
+										<echo message="Skipping native platform @{platform} because we are running an a ${this_platform} machine" />
```

Comment:
I won't be pointing out too many typos in this PR, but "running on a" vs "running an a"

Reply:
Fixed.